### PR TITLE
Relaybot startup fixes

### PR DESCRIPTION
**Pull request by AndrewFerr**
This should fix the crash on relaybot startup introduced by [v0.12.1-mod-1](https://github.com/vector-im/mautrix-telegram/releases/tag/v0.12.1-mod-1).

Commits are ordered from most to least important (age ∝ importance).

### CODE REVIEW

File: mautrix_telegram/__main__.py [hunk 1]
```diff
@@ -121,8 +121,6 @@ def prepare_bridge(self) -> None:
         # Note: In upstream this would start all the puppets, but in our fork we want to gracefully start the user puppets
         self.add_startup_actions(User.init_cls(self))
         self.add_startup_actions(Portal.restart_scheduled_disappearing())
-        if self.bot:
-            self.add_startup_actions(self.bot.start())
```
Comment (buffless-matt) [inline mautrix_telegram/__main__.py:125]: Presumably the bot start is still happening once (somewhere else)?

**Review by buffless-matt — APPROVED** [review]
Looks sane, but I'd recommend a bridge team member give it a once-over as well.

File: mautrix_telegram/__main__.py (hunk 1 above)
Comment (AndrewFerr) [reply mautrix_telegram/__main__.py:125]: Yes, on line 155

**Review by jaller94 — APPROVED** [review]
Looks sane. +1

**Comment by jaller94** [comment]
This did not solve the error.

**Comment by AndrewFerr** [comment]
This did solve the error after all -- the new error was a different one.  I've put an explanation of it in the issue.

**Pull request merged**